README.md

```diff
@@ -1164,6 +1164,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Siddhant Khisty](https://github.com/SiddhantKhisty)
   - [Siddhant Ota](https://github.com/sid00100)
   - [Siddharth Bihani](https://github.com/siddharthbihani)
+  - [Siddharth Rai](https://github.com/settings/profile)
```

Comment:
Please provide your GitHub profile link here.

Please let us know if you need any help!

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix